components/channel_layout/app_route/app_route.jsx

```diff
@@ -31,7 +30,7 @@ export default class AppRouter extends React.PureComponent {
         if (this.props.channelId) {
             this.props.actions.selectChannel(this.props.channelId);
         }
-        $('body').addClass('app__body');
+        document.querySelector('body').classList.add('app__body');
```

Comment:
I think you should be using here `document.body` directly, is more clean.

Reply:
done